lib/runtime/wasmer/config.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 ChainSafe Systems (ON)
+// Copyright 2023 ChainSafe Systems (ON)
```

Comment:
I think we should keep the original date when the file was created

Reply:
Ah okay I wasn't sure, will update